plugins/BEdita/Core/src/Model/Table/ObjectTypesTable.php

```diff
@@ -197,10 +197,13 @@ public function get($primaryKey, $options = [])
             $primaryKey = $allTypes[$primaryKey];
         }
 
-        $options += [
-            'key' => 'id_' . $primaryKey,
-            'cache' => self::CACHE_CONFIG,
-        ];
+        if (empty($options)) {
+            $options = [
+                'key' => 'id_' . $primaryKey,
```

Comment:
this is minor minor minor .... since this cache contains relations information mabe a better name could be `related_1` or `1_related` ?